beetsplug/web/__init__.py

```diff
@@ -337,6 +338,11 @@ def func(lib, opts, args):
                     r"/*": {"origins": self.config['cors'].get(str)}
                 }
                 CORS(app)
+
+            if self.config['subsonic']:
+                from .subsonic import subsonic_routes
+                app.register_blueprint(subsonic_routes)
+
```

Comment:
And then eventually the AURA stuff could just be implemented as a separate plugin namespaced under `/aura/`.